src/rdata.cpp

```diff
@@ -444,6 +445,21 @@ void ReturnData::handleSx0Backward(const Model &model,
     auto xQBpreeq = preeq.getAdjointQuadrature();
     for (int ip = 0; ip < model.nplist(); ++ip)
         xQB[ip] += xQBpreeq[ip];
+
+    /* Prefer explicit type declaration here, since this MUST be a reference
+     * (for memory reasons), rather than passing this thing by value */
```

Comment:
Not sure if more helpful or confusing. Just taking the reference here would be clear enough for me.

Reply:
changed it